{
	"schema_version": "1.6.0",
	"id": "CVE-2022-48695",
	"modified": "2024-09-18T03:22:33.468968Z",
	"published": "2024-05-03T18:15:08Z",
	"details": "In the Linux kernel, the following vulnerability has been resolved:\n\nscsi: mpt3sas: Fix use-after-free warning\n\nFix the following use-after-free warning which is observed during\ncontroller reset:\n\nrefcount_t: underflow; use-after-free.\nWARNING: CPU: 23 PID: 5399 at lib/refcount.c:28 refcount_warn_saturate+0xa6/0xf0",
	"affected": [
		{
			"package": {
				"ecosystem": "Debian:11",
				"name": "linux",
				"purl": "pkg:deb/debian/linux?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "5.10.148-1"
						}
					]
				}
			],
			"versions": [
				"5.10.103-1",
				"5.10.103-1~bpo10+1",
				"5.10.106-1",
				"5.10.113-1",
				"5.10.120-1",
				"5.10.120-1~bpo10+1",
				"5.10.127-1",
				"5.10.127-2",
				"5.10.127-2~bpo10+1",
				"5.10.136-1",
				"5.10.140-1",
				"5.10.46-4",
				"5.10.46-5",
				"5.10.70-1",
				"5.10.70-1~bpo10+1",
				"5.10.84-1",
				"5.10.92-1",
				"5.10.92-1~bpo10+1",
				"5.10.92-2"
			],
			"ecosystem_specific": {
				"urgency": "not yet assigned"
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2022-48695.json"
			}
		},
		{
			"package": {
				"ecosystem": "Debian:12",
				"name": "linux",
				"purl": "pkg:deb/debian/linux?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "5.19.11-1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"urgency": "not yet assigned"
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2022-48695.json"
			}
		},
		{
			"package": {
				"ecosystem": "Debian:13",
				"name": "linux",
				"purl": "pkg:deb/debian/linux?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "5.19.11-1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"urgency": "not yet assigned"
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2022-48695.json"
			}
		}
	],
	"references": [
		{
			"type": "WEB",
			"url": "https://git.kernel.org/stable/c/41acb064c4e013808bc7d5fc1b506fa449425b0b"
		},
		{
			"type": "WEB",
			"url": "https://git.kernel.org/stable/c/5682c94644fde72f72bded6580c38189ffc856b5"
		},
		{
			"type": "WEB",
			"url": "https://git.kernel.org/stable/c/6229fa494a5949be209bc73afbc5d0a749c2e3c7"
		},
		{
			"type": "WEB",
			"url": "https://git.kernel.org/stable/c/82efb917eeb27454dc4c6fe26432fc8f6c75bc16"
		},
		{
			"type": "WEB",
			"url": "https://git.kernel.org/stable/c/991df3dd5144f2e6b1c38b8d20ed3d4d21e20b34"
		},
		{
			"type": "WEB",
			"url": "https://git.kernel.org/stable/c/b8fc9e91b931215110ba824d1a2983c5f60b6f82"
		},
		{
			"type": "WEB",
			"url": "https://git.kernel.org/stable/c/d4959d09b76eb7a4146f5133962b88d3bddb63d6"
		},
		{
			"type": "WEB",
			"url": "https://git.kernel.org/stable/c/ea10a652ad2ae2cf3eced6f632a5c98f26727057"
		},
		{
			"type": "ADVISORY",
			"url": "https://security-tracker.debian.org/tracker/CVE-2022-48695"
		}
	]
}
